css-layout-classes/styles.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+img {
+  width: 100%;
+}
+
+h1,
+h2 {
+  font-family: 'Roboto', sans-serif;
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+}
+
+.row {
+  display: flex;
+
+  /* flex-wrap: wrap; */
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+#two-thirds {
+  width: 70%;
+}
+
+#one-third {
+  width: 30%;
+}
+
+#two-thirds2 {
+  width: 70%;
+}
+
+#one-third2 {
+  width: 30%;
+}
```

Comment:
Instead of using an `id` on these selectors here, you should make a class that you can just reuse.